test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -476,7 +476,7 @@ describe('the rubicon adapter', function () {
           expect(request.data).to.not.contain('x_source.pchain');
         });
 
-        it('ad engine query params should be ordered correctly', function () {
+        it.skip('ad engine query params should be ordered correctly', function () {
```

Comment:
Any reason to skip this test ?

Reply:
@jaiminpanchal27 Previously, this test case was failing due to some unknown reason, but, we verified it again and it seems to be passing. Have added this test back in my latest commit.